packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -290,7 +290,7 @@ class LongPressDraggable<T> extends Draggable<T> {
       ..onStart = (Offset position) {
         final Drag result = onStart(position);
         if (result != null)
-          HapticFeedback.vibrate();
+          HapticFeedback.selectionClick();
```

Comment:
This changes the Android implementation from using `HapticFeedbackConstants.LONG_PRESS` to `HapticFeedbackConstants.CLOCK_TICK`.  An alternative would be to check the platform and dispatch a selectionClick on iOS and fallback to a vibrate otherwise

Reply:
This level of feedback is actually more consistent with the other draggables that I've grabbed on Android: the reorderable language dialog seems to use the same level of feedback, as well as the app reorder action on the home screen.